install/newinstall.rst

```diff
@@ -218,8 +212,7 @@ See :ref:`newinstall-user-python` if required.
 The Python environment installed by :command:`newinstall.sh` is Miniconda_, a minimal version of Anaconda_.
 By default, :command:`newinstall.sh` installs Python 3.6.
```

Comment:
I think it's 3.7 now.

Reply:
Changed wording to avoid referring to a specific version number.